scraping/main.py

```
@@ -171,6 +175,7 @@ def update_stats_in_transaction() -> None:
                             )
                     except Post.DoesNotExist:
                         logger.warning(f"Post not found: {post_id}")
+                        sentry_sdk.capture_exception(e)
```

Ruff (0.8.2)

Undefined name `e`

(F821)